deltacat/compute/compactor_v2/model/merge_input.py

```diff
@@ -57,9 +63,10 @@ def of(
         result["read_kwargs_provider"] = read_kwargs_provider
         result["round_completion_info"] = round_completion_info
         result["object_store"] = object_store
+        result["delete_file_envelopes"] = delete_file_envelopes
+        result["delete_strategy"] = delete_strategy
```

Comment:
The `DeltaFileEnvelopes` were refactored to be provided by `MergeFileGroupsProvider`, should we also extend this provider to include `deleteFileEnvelopes` as well

Reply:
`DeleteFileEnvelopes` never touch the hash-bucket step so I don't think this is necessary/in-scope.

Reply:
The intention of `DeleteFileEnvelopes` subclassing from `DeltaFileEnvelopes` is to ensure that can create list of type List[DeltaFileEnvelopes] within the merge step to handle deletes and sort it appropriately

Reply:
Sounds good